<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagSettingDialog</class>
 <widget class="QDialog" name="DiagSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diagnostic Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Enable Diagnostic Analysis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>&amp;vc fsm export</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>vc fsm export file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEditFile"/>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>report measured safe:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBoxRMS">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>report dc krf:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>report dc kmpf:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="comboBoxKmpf">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="comboBoxKrf">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>undetected faults:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUndetectedFaults">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>assumed_safe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>assumed_dangerous</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Failure Mode:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetFailureMode">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id label</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>&lt;Click to add&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Measurement:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetMeas">
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>check</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>meas name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>min</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>max</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>category</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>&lt;Click to add&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_7</zorder>
  <zorder>tableWidgetMeas</zorder>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
  <zorder>label</zorder>
  <zorder>tableWidgetFailureMode</zorder>
  <zorder>checkBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiagSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiagSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
